src/components/Jobs/Jobs.tsx

```diff
@@ -132,7 +134,7 @@ function Jobs() {
   const { onToggleHandler, filter } = useEntityFilter(ALL_JOBS, MY_JOBS);
   const shouldShowAllJobs = filter !== MY_JOBS;
   const { user } = useAuth();
-  const { data = [] } = useJobs();
+  const { data = [], refetch } = useJobs();
```

Comment:
Instead of calling refetch, I was thinking maybe adding a useMutation hook in the useJobs hook file which calls the killJobs method in the service file and then invalidates query to fetch all jobs.

See https://react-query-v3.tanstack.com/reference/useMutation

Reply:
Got it !!